accord-core/src/main/java/accord/local/Node.java

```diff
@@ -203,7 +203,7 @@ public void withEpoch(long epoch, Runnable runnable)
         else
         {
             configService.fetchTopologyForEpoch(epoch);
-            topology.awaitEpoch(epoch).addCallback(runnable);
+            topology.awaitEpoch(epoch).addCallback(runnable).begin(agent());
```

Comment:
I changed the type returned from `awaitEpoch` to be `AsyncChain`, so need to "begin" it.

The reason is that `AsyncResult` doesn't know what `CommandStore` to run in, so if you are in a `CommandStore` you need to start a new chain, so something has to `begin` it